src/Codeception/Test/Descriptor.php

```diff
@@ -32,15 +32,22 @@ public static function getTestSignature(\PHPUnit\Framework\SelfDescribing $testC
      */
     public static function getTestSignatureUnique(\PHPUnit\Framework\SelfDescribing $testCase)
     {
+        $env     = null;
```

Comment:
Empty string would be a better choice for default value,
You are only using it for string concatenation.